logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/callback/ParticipantChangedHandlerImpl.kt

```diff
@@ -0,0 +1,55 @@
+package com.wire.kalium.logic.feature.call.callback
+
+import com.sun.jna.Pointer
+import com.wire.kalium.calling.callbacks.ParticipantChangedHandler
+import com.wire.kalium.logic.data.call.AvsClient
+import com.wire.kalium.logic.data.call.AvsClientList
+import com.wire.kalium.logic.data.call.Participant
+import com.wire.kalium.logic.data.id.QualifiedID
+import com.wire.kalium.logic.util.getDomain
+import com.wire.kalium.logic.util.removeDomain
+import org.json.JSONObject
+
+class ParticipantChangedHandlerImpl(
```

Comment:
I'm thinking a lot of this kind of Handlers are platform-agnostic and could be moved to `commonMain` somewhere in the not-so-far future.

In any case, we could give them the blessing of testing.

Reply:
Not really sure we could move some of the handler/callbacks implemented due to them receiving a `Pointer` 🤔